cmd/currency-service/internal/config.go

```diff
@@ -0,0 +1,26 @@
+package internal
+
+import (
+	"subscription-api/config"
+
+	"github.com/caarlos0/env/v9"
+)
+
+type envVars struct {
+	Mode                   config.Mode `env:"MODE"`
+	Port                   string      `env:"CURRENCY_SERVICE_PORT"`
+	CurrencyServiceServer  string      `env:"CURRENCY_SERVICE_SERVER"`
+	ExchangeCurrencyAPIKey string      `env:"EXCHANGE_CURRENCY_API_KEY"`
+}
+
+var vars envVars
```

Comment:
Also bad practice is singleton variables

Reply:
https://refactoring.guru/design-patterns/singleton/go/example

Reply:
Got rid of all singletons in project.